test/TesterInternal/SchedulerTests/QueuedTaskSchedulerTests_Set2.cs

```diff
@@ -1,587 +0,0 @@
-        {
-            string testName = "Task_OneSchedulerMultiContexts";
-
-            var baseline = DoBaseTestRun(testName + "-Baseline", numTasks);
-
-            var tasks = new List<Task>(numTasks);
-
-            var taskScheduler = GetTaskScheduler();
-
-            SynchronizationContext[] syncContexts = new SynchronizationContext[numTasks];
-            for (int i = 0; i < numTasks; i++)
-            {
-                syncContexts[i] = new AsyncTestContext(output);
-            }
-
-            QueuedTaskSchedulerTests_Set1.TimeRun(1, baseline, testName, output, () =>
-            {
-                for (int i = 0; i < numTasks; i++)
-                {
-                    Task t = CreateTask(i);
-                    SynchronizationContext.SetSynchronizationContext(syncContexts[i]);
-                    t.Start(taskScheduler);
-                    tasks.Add(t);
-                }
-
-                Task.WaitAll(tasks.ToArray());
-            });
-
-            foreach (Task t in tasks)
-            {
-                Assert.IsTrue(t.IsCompleted, "Task is completed");
-                Assert.IsFalse(t.IsFaulted, "Task did not fault");
-                Assert.IsNull(t.Exception, "Task did not return an Exception");
-            }
-        }
-
-        [Fact, TestCategory("Scheduler"), TestCategory("Tasks"), TestCategory("TaskScheduler")]
-        public void Task_MultiSchedulerMultiContexts()
-        {
-            string testName = "Task_MultiSchedulerMultiContexts";
-
-            var baseline = DoBaseTestRun(testName + "-Baseline", numTasks);
-
-            var tasks = new List<Task>(numTasks);
-
-            SynchronizationContext[] syncContexts = new SynchronizationContext[numTasks];
-            for (int i = 0; i < numTasks; i++)
-            {
-                syncContexts[i] = new AsyncTestContext(output);
-            }
-            TaskScheduler[] schedulers = new TaskScheduler[numTasks];
-            for (int i = 0; i < numTasks; i++)
-            {
-                schedulers[i] = GetTaskScheduler();
-            }
-
-            QueuedTaskSchedulerTests_Set1.TimeRun(1, baseline, testName, output, () =>
-            {
-                for (int i = 0; i < numTasks; i++)
-                {
-                    Task t = CreateTask(i);
-                    SynchronizationContext.SetSynchronizationContext(syncContexts[i]);
-                    t.Start(schedulers[i]);
-                    tasks.Add(t);
-                }
-
-                Task.WaitAll(tasks.ToArray());
-            });
-
-            foreach (Task t in tasks)
-            {
-                Assert.IsTrue(t.IsCompleted, "Task is completed");
-                Assert.IsFalse(t.IsFaulted, "Task did not fault");
-                Assert.IsNull(t.Exception, "Task did not return an Exception");
-            }
-        }
-
-        [Fact, TestCategory("Scheduler"), TestCategory("Tasks"), TestCategory("TaskScheduler")]
-        public void Task_OrleansTaskScheduler()
```

Comment:
The only test that uses `OrleansTaskScheduler` instead of `QueuedTaskScheduler`. If this one is still needed, it can be restored.